deploy/fb-av-chart/templates/deployment.yaml

```diff
@@ -4,7 +4,7 @@ kind: Deployment
 metadata:
   name: "fb-av-{{ .Values.environmentName }}"
 spec:
-  replicas: 2
+  replicas: 1
```

Comment:
Why 1?

Reply:
its a pretty beefy box, don't see why we need 2 of them at the moment